### ISSUE TITLE
win2012r2-standardcore-ssh.json template has the wrong vm_name for some of the builders

### ISSUE THREAD

**Issue by arizvisa** [boxcutter/windows#214]
While checking the `vm_name` for some of the templates, the `win2012r2-standardcore-ssh.json` template is using the same name for the `win2012r2-standardcore.json` template. This inconsistency will only affect the template name during building, but prior to post-processing the template into a vagrant box. Things can get funky if you're building both of these templates at the same time from the same directory.

**Comment by arizvisa** [comment]
Ftr this only affects the `vmware-iso`, `virtualbox-iso`, and `parallels-iso` builders.

This was found with the following `jq` query:
```
jq '([.builders[] | .vm_name] | unique) as $names | {name: $names, length: ($names | length) } | select(.length > 1)' *.json
```

**Comment by arizvisa** [comment]
This is fixed by PR #215.
